infra/terraform/dev-org/dev-project-cleanup/versions.tf

```diff
@@ -15,25 +15,22 @@
  */
 
 terraform {
-  required_version = ">= 0.12"
-}
-
-provider "archive" {
-  version = "~> 1.3"
-}
-
-provider "google" {
-  version = "~> 3.35"
-}
-
-provider "google-beta" {
-  version = "~> 3.35"
-}
-
-provider "null" {
-  version = "~> 2.1"
-}
-
-provider "random" {
-  version = "~> 2.2"
+  required_version = ">= 0.13"
+  required_providers {
+    archive = {
+      version = "~> 1.3"
+    }
+    google = {
+      version = ">= 3.35, < 5.0"
```

Comment:
Is there a reason you're doing < 5.0 for the provider?

Reply:
That's in case there is breaking change in TPG 5, but I'm happy to remove from this project if that is preferred?

Reply:
I don't think we use that convention elsewhere. I would suggest removing it. The PR is good to go otherwise.

Reply:
We should keep the major version constraint for providers.